week7/cooper_client_react/04_saving_data.md

```diff
@@ -0,0 +1,252 @@
+Next feature that we want to implement is that the user can save their cooper result.
```

Comment:
A summary here as well would be good
and an explanation on why we are doing this now instead of in the first step, like we work in an incremental way.. osv osv